### DISCUSSION TITLE
Didn't Received Student Benefits

### DISCUSSION THREAD

**Discussion by saikannadkar** [community/community#146812] [GitHub Education] [answered]
### Hi everyone,

I have Been approved for Student Developer Pack but still didn't received the Benefits.

**Comment by YifeiZhang123** [comment]
me too,and whats is the reason

**Reply by YifeiZhang123** [reply]
Man, I just read the regulations, I think there should be a buffer time between the two

**Comment by davevad93** [comment, marked as answer]
Hi @saikannadkar ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @saikannadkar was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by YifeiZhang123** [reply]
I desperately need to use Copilot, can you help me faster?
